code/go/0chain.net/smartcontract/storagesc/readpool.go

```diff
@@ -104,15 +107,15 @@ func (rp *readPool) moveToBlobber(sscKey, allocID, blobID string,
 		}
 		var (
 			bp   = ap.Blobbers[bi]
-			move state.Balance
+			move currency.Coin
 		)
 		if value >= bp.Balance {
 			move, bp.Balance = bp.Balance, 0
 		} else {
 			move, bp.Balance = value, bp.Balance-value
```

Comment:
the move should be value - previous moved, so should the bp.Balance be bp.Balance - value + previous moved